src/clj_kondo/core.clj

```diff
@@ -21,8 +21,10 @@
 (defn print!
   "Prints the result from `run!` to `*out*`. Returns `nil`. Alpha,
   subject to change."
-  [{:keys [:config :findings :summary :analysis]}]
+  [{:keys [:config :findings :summary :analysis]} report-level]
```

Comment:
Since this is a public API function adding another argument would be a breaking change to existing caller. Just make :report-level another option in the map passed as the argument.